Lib/test/test_asyncio/test_proactor_events.py

```diff
@@ -740,7 +744,9 @@ def test_close_self_pipe(self):
 
     def test_close(self):
         self.loop._close_self_pipe = mock.Mock()
-        self.loop.close()
+        with mock.patch('signal.set_wakeup_fd') as set_wakeup_fd:
+            set_wakeup_fd.return_value = -1000
+            self.loop.close()
```

Comment:
I guess this test needs to row an `assertWarnsRegex` call? (See my comment on the code in the file under test.)

Reply:
Yeap, that's right too... This is an incredibly sloppy patch. It shows I did it on a rush. Sorry for that. Fixed in https://github.com/python/cpython/pull/96807/commits/bc7b849d3b29b643f0352441142e95fb0a72bcbe.